modules/dynamic-operator/config/manager/manager.yaml

```diff
@@ -128,6 +128,11 @@ spec:
                 secretKeyRef:
                   name: kuberlogic-config
                   key: SENTRY_DSN
+            - name: KUBERLOGIC_DOMAIN
```

Comment:
Should be added to the install operation as well

Reply:
As I see it's already added there